setup.py

```diff
@@ -55,7 +55,7 @@ def version():
     long_description_content_type="text/markdown",
     author="Yaron Haviv",
     author_email="yaronh@iguazio.com",
-    license="MIT",
+    license="Apache License Version 2.0",
```

Comment:
I think that it should be:
```suggestion
    license="Apache-2.0",
```
https://spdx.org/licenses/Apache-2.0.html

Reply:
id go with full name (`Apache License 2.0`)